lib/src/page_list_viewport.dart

```diff
@@ -406,6 +406,7 @@ class PageListViewportController with ChangeNotifier {
           case AnimationStatus.dismissed:
           case AnimationStatus.completed:
             _velocity = Offset.zero;
+            notifyListeners();
```

Comment:
I'm wondering if we should be notifying listeners in the regular listener, which would notify on every frame? Is there a reason that we only care at the end of the animation, rather than during every change?